--- origsrc/muine-0.8.9/doc/Makefile.am	2008-07-04 09:11:27.000000000 -0500
+++ src/muine-0.8.9/doc/Makefile.am	2008-07-28 23:06:02.156250000 -0500
@@ -5,8 +5,8 @@
 
 ASSEMBLIES_BUILD = $(foreach asm,$(ASSEMBLIES),$(addprefix -assembly:,$(asm)))
 
-docdir = $(libdir)/monodoc/sources
-doc_DATA = \
+monodocdir = $(libdir)/monodoc/sources
+monodoc_DATA = \
 	muine-docs.zip \
 	muine-docs.tree \
 	muine-docs.source
--- origsrc/muine-0.8.9/libmuine/Makefile.am	2008-07-04 09:11:26.000000000 -0500
+++ src/muine-0.8.9/libmuine/Makefile.am	2008-07-28 22:19:12.921875000 -0500
@@ -53,3 +53,4 @@
 	mm-keys.h
 	
 libmuine_la_LIBADD = $(id3_vfs_lib) $(MUINE_LIBS) $(OGG_LIBS) $(VORBIS_LIBS) $(VORBISFILE_LIBS) $(GDBM_LIBS) $(ID3TAG_LIBS) $(FLAC_LIBS) $(FAAD_LIBS)
+libmuine_la_LDFLAGS = -module -avoid-version -no-undefined
--- origsrc/muine-0.8.9/src/muine.exe.config.in	2008-07-04 09:11:26.000000000 -0500
+++ src/muine-0.8.9/src/muine.exe.config.in	2008-07-28 22:19:12.921875000 -0500
@@ -1,8 +1,8 @@
 <configuration>
-  <dllmap dll="libglib-2.0-0.dll" target="libglib-2.0.so.0"/>
-  <dllmap dll="libgtk-2.0-0.dll" target="libgtk-x11-2.0.so.0"/>
-  <dllmap dll="libgobject-2.0-0.dll" target="libgobject-2.0.so.0"/>
-  <dllmap dll="libgnomevfs-2-0.dll" target="libgnomevfs-2.so.0"/>
-  <dllmap dll="musicbrainz" target="libmusicbrainz.so.4"/>
-  <dllmap dll="libmuine" target="@pkglibdir@/libmuine.so"/>
+  <dllmap dll="libglib-2.0-0.dll" target="cygglib-2.0-0.dll"/>
+  <dllmap dll="libgtk-2.0-0.dll" target="cyggtk-x11-2.0-0.dll"/>
+  <dllmap dll="libgobject-2.0-0.dll" target="cyggobject-2.0-0.dll"/>
+  <dllmap dll="libgnomevfs-2-0.dll" target="cyggnomevfs-2-0.dll"/>
+  <dllmap dll="musicbrainz" target="cygmusicbrainz-4.dll"/>
+  <dllmap dll="libmuine" target="@pkglibdir@/cygmuine.dll"/>
 </configuration>
